docs/mine-hnt/validators/requirements.mdx

```diff
@@ -13,13 +13,23 @@ Based on early tests, validator node requirements include:
   - These recommendations may change for mainnet as the size of consensus groups grow and hotspot transactions are added
 - Memory: 8 GB or above
-- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet.
-- If using AWS, a t2.large or t2.xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with similar capabilities should be the target.
+- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet. This should not be on the OS disk and remoting your `/home/*` directory on a seperate disk is recommended.
+- If using AWS, a t2.xlarge or c5a.xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with similar capabilities should be the target.  
 - Static IP address with ports `2154`(TCP) and `8080`(TCP) opened in your firewall with a route available to your validator host.
 - Running on stable network connection free of things like proxies, NAT, etc. The load is largely symmetrical when producing blocks, so good upstream connectivity recommended.
 - It is [discouraged to run Validators at home](/mine-hnt/validators/faqs-resources-providers/#technical). The one exception to this is if you live at an actual data center. If that's the case, you're allowed to run Validators at home.
+
+- Overall Baselining (do this after the validator is caught up to the tip of the chain, and running normally for a few hours)
+  - Review the Underlying Hardware your provider is using You can review your instance cpu speeds by using `cat /proc/cpuinfo | egrep  -i "name|stepping|Mhz" | sort | uniq -c` Some providers use the Turbo speeds as marketing and others the regular speed of the processor.
+  - You can review your CPU steal using `sar` and looking at the `%steal` column. Averages of over 1% you should think about moving providers. You can also see this metric in `top` look for the `st` cpu metric on the second line.
+  - You can review your IOPS using `sar -b` 
+  - You can review your Disk Bandwidth using `sar -d -p | egrep "sd[a-z]|DEV"`
+  - You can review your Disk Latency with `sar -d -p | egrep "sd[a-z]|DEV"` review the `await` which should be < 10ms
+  - Review your Absorb and Commit Block times and they should be < 2500ms on average, not including the rewards blocks that will be much higher. To check ongoing times you can use `tail -f log/console.log | grep -E 'add_block|absorb_and_commit'`
+  - Once your validator is caught up you can use `grep -E 'add_block|absorb_and_commit' log/console.log | awk '{print $6 $8 }' | sed ':a;N;$!ba;s/\ngossipped/ /g'` as a historical listing of your console.log
+  - If you believe you are having trouble getting the advirtised disk performance you can install and run the `fio` package to give you a full baseline of disk performance.
```

Comment:
You might want to warn people not to benchmark a validator that is currently in CG, or that might be really soon.

Reply:
Agreed